src/components/dashboard/StatusPanel.js

```diff
@@ -8,6 +8,8 @@ import * as statusService from '../../services/status';
 
 import Panel from '../commons/Panel';
 import ServiceList from './ServiceList';
+import LogList from './LogList';
+
```

Comment:
Extra line . And make sure you add imported files in proper order . 
.
..
...
....


```
import LogList from './LogList';
import ServiceList from './ServiceList';
import Panel from '../commons/Panel';
```